pages/[address].tsx

```diff
@@ -166,10 +195,37 @@ const SynthPage: React.FC<Props> = ({ data, relatedSynths, change24h }) => {
     ["synth state", data.address],
     async () => await client.getState<typeof data>(data.address)
   );
+  const { data: synthStats } = useQuery(
+    ["synth stats", data.address],
+    async () => await client.getSynthStats(data.address)
+  );
   const { data: tvlHistory, isLoading: isLoadingTvl } = useQuery(
     ["tvl history", data.address],
     async () => await client.getTvl(data.address)
   );
+
+  const { data: change24h } = useQuery(
+    ["tvl change", data.address],
+    async () => {
+      const lastTvl = await client.getLatestTvl(data.address);
+      const [{ value: ydayTvl = NaN } = {}] = await client.request(
+        "global/tvlHistorySlice",
+        data.address,
+        Math.floor(oneDayAgo().toSeconds())
+      );
+      const tvl24hChange = !Number.isNaN(ydayTvl)
+        ? Math.round(
+            ((formatWeiString(lastTvl) - formatWeiString(ydayTvl)) /
+              formatWeiString(ydayTvl)) *
+              1000
+          ) / 10
```

Comment:
If we are dealing with Big Numbers here, we should be doing this math using the ethers BigNumber library.

Reply:
I agree with this in general, but this is just TVL change so we should be fine

Reply:
i think its ok to treat this particular number with low precision